docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -95,7 +95,7 @@ export default function App() {
 }
 ```
 
-{{"demo": "pages/components/data-grid/getting-started/Codesandbox.js", "hideToolbar": true, "bg": true}}
+{{"demo": "pages/components/data-grid/getting-started/Codesandbox.js", "bg": true}}
```

Comment:
🚧``
```suggestion
{{"demo": "pages/components/data-grid/getting-started/Codesandbox.js", "hideToolbar": true, "bg": true}}
```

Reply:
So this should be `"showToolbar": false` ?

Reply:
This is unrelated, it's the layout of the documentation.

Reply:
ahhaha omg, sorry 😅  It's because I did a find and replace for `hideToolbar`.